File: twoSum/twoSum.java
class Solution {
  public static int[] twoSum(int[] nums, int target) {
    int[] anwsers = new int[2];
    for (int i = 0; i < nums.length; i++) {
      for (int j = nums.length - 1; j >= 0; j--) {
        if (i == j) {
          continue;
        }
        if (nums[i] + nums[j] == target) {
          anwsers[0] = i;
          anwsers[1] = j;
          return anwsers;
        }
      }
    }
    return anwsers;
  }
}

File: first/twoSum/twoSum.ts
/**
 * O(n2)暴力求解
 */

function twoSum(nums: number[], target: number): number[] {
  let anwser: number[] = [];
  for (let i = 0; i < nums.length; i++) {
    for (let j = nums.length - 1; j >= 0; j--) {
      if (i === j) {
        continue
      }
      if (nums[i] + nums[j] === target) {
        anwser[0] = i;
        anwser[1] = j;
        return anwser;
      }
    }
  }
  return anwser;
}

